api/models/Supplier.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+
+public class Supplier : BaseModel
+{
+    public Supplier() { }
+    public Supplier(bool newInstance = false, bool isUpdate = false) : base(newInstance, isUpdate) { }
+    public string? Code { get; set; }
+    public string? Name { get; set; }
+    public string? Reference { get; set; }
+    // foreing key
+    public Guid ContactId { get; set; }
+    public Contact? Contact { get; set; }
+    public Guid AddressId { get; set; }
+    public Address? Address { get; set; }
```

Comment:
Zorg ervoor dat de attributen de required fields hebben net als in de excelsheet

![image](https://github.com/user-attachments/assets/1da17f5a-1030-4fc2-9065-e91ba6ce02d5)